src/Symfony/Component/Lock/Factory.php

```diff
@@ -36,14 +36,15 @@ public function __construct(StoreInterface $store)
     /**
      * Creates a lock for the given resource.
      *
-     * @param string $resource The resource to lock
-     * @param float  $ttl      maximum expected lock duration
+     * @param string $resource    The resource to lock
+     * @param float  $ttl         Maximum expected lock duration in seconds
+     * @param bool   $autoRelease Whether to automatically release the lock or not
```

Comment:
Whether to automatically release the lock or not when the lock instance is destroyed